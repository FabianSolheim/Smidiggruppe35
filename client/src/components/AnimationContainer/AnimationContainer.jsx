import React from "react";
import { motion } from "framer-motion";
import { container } from "../../styling/animation/framerAnimations";

const AnimationContainer = ({ children }) => {
  return (
    <motion.div
      initial="pageInitial"
      animate="pageAnimate"
      variants={container}
    >
      {children}
    </motion.div>
  );
};

export default AnimationContainer;
