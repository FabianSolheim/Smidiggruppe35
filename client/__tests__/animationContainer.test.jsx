import React from "react";
import ReactDOM from "react-dom";
import AnimationContainer from "../src/components/AnimationContainer/AnimationContainer";

describe("AnimationContainer test", () => {
  it("shows AniamtionContainer", () => {
    const domElement = document.createElement("div");
    ReactDOM.render(<AnimationContainer />, domElement);
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
