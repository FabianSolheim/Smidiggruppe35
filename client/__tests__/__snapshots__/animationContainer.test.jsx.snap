// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimationContainer test shows AniamtionContainer 1`] = `"<div style=\\"opacity: 0;\\"></div>"`;
